### ISSUE TITLE
Middle Cerebral Artery berry aneurysm

### ISSUE THREAD

**Issue by albodrug** [obophenotype/human-phenotype-ontology#11454]
**Preferred term label:**
Middle Cerebral Artery berry aneurysm

**Synonyms**
MCA berry aneurysm

**Definition (free text, please give PubMed ID)**
PMID: [29304371](https://pubmed.ncbi.nlm.nih.gov/29304371/) "IA is located on the middle cerebral artery bifurcation in 7 case subjects (58%), on the anterior communicant artery, the anterior cerebral artery, and the internal carotid artery in 3 case subjects (25%), and on the posterior communicant artery in 2 case subjects (17%)."
PMID: 6474536 "We classified aneurysms according to their sites as follows: 1) Aneurysms of ACA (ACA an) including those of proximal and distal segments of ACA but not including those of ACA-AComA junctions, 2) aneu- rysms of AComA (AComA an) including all aneurysms of ACA-AComA junctions, 3) aneurysms of the internal carotid artery-PComA junctions (ICA-PComA an), 4) aneurysms of ICA-anterior choroidal artery junctions (ICA-AChA an), 5) aneurysms of AChA (AChA an), 6) aneurysms of ICA bifurcation (ICA Bif an), 7) aneurysms of MCA (MCA an), 8) aneurysms of the basilar artery (BA an), and 9) aneurysms of VA (VA an). "

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0007029

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
could characterize:
MONDO:0005291 A congenital or acquired aneurysm within the cranium.
ORPHA:231160 Familial cerebral saccular aneurysm

**Your nano-attribution (ORCID)**
0009-0006-3873-102X

**Comment by pnrobinson** [comment]
Term added as HP_0020316
